### DISCUSSION TITLE
Solution Architecture Patterns 0.95 Released!

### DISCUSSION THREAD

**Discussion by chanakaudaya** [chanakaudaya/solutions-architecture-patterns#29] [General]
# Release Notes
This is the latest release of the Solutions Architecture Patterns repository. This release contains several new architecture patterns as well as contributions from new contributors.

### Vendor-neutral architecture patterns
- Change Data Capture pattern
- Innovations Driven Enterprise Pattern

### Industry-specific architecture patterns
- Higher Education Industry IT Reference Architecture
- Energy Industry IT Reference Architecture
- Automotive Industry IT Reference Architecture

### Technology selection guides
- Integration Platform selection guide

#### WSO2
- WSO2 Deployment Reference Architecture on AWS
- WSO2 Deployment Reference Architecture on Azure

## Improvements
- Added a new section with related technology architecture GitHub repository links
- Changed the license to CC By 4.0 to make the content free to use

# Contributing
We are looking for contributors who can contribute with their experiences in the solution architecture domain. Send your PR to the repository.

<hr /><em>This discussion was created from the release <a href='https://github.com/chanakaudaya/solution-architecture-patterns/releases/tag/0.95'>Solution Architecture Patterns 0.95 Released!</a>.</em>

**Comment by adityardeshpande** [comment]
Thank you for putting this together. This is wonderful stuff!
